ImgShareData/ImgurEnum.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImgShare.APISource.Data
+{
+    public enum ImgurParameters

```

Comment:
nit (well less nit than a regular nit but still nit): these are public and have no xmldoc! That's makes me sad :cry: